test/subgraph-tests/orders.test.ts

```
@@ -1,19 +1,33 @@
 
   it('should return an order using the order Id', async () => {
     const parifiSdk = await getParifiSdkInstanceForTesting();
-    const orderId = 'CLOSEDORD10209728236255228114-100-0';
-    const order = await parifiSdk.subgraph.getOrderById(orderId);
-    expect(order.id).toBe(orderId);
+    const order = await parifiSdk.subgraph.getOrderById(TEST_ORDER_ID1);
+    expect(order.id).toBe(TEST_ORDER_ID1);
     expect(order.status).toBe(status.SETTLED);
   });
+
+  it.only('should return an settled order count for a user', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)